services/galley/src/Galley/API/Teams/Features.hs

```diff
@@ -496,6 +497,11 @@ getSelfDeletingMessagesInternal = \case
       TeamFeatures.getSelfDeletingMessagesStatus tid
         <&> maybe Public.defaultSelfDeletingMessagesStatus id
 
+setSelfDeletingMessagesPaymentUnlockInternal ::
```

Comment:
we may want to keep this definition for symmetry with the rest of the code, but call it from `iUnlockPayment` directly instead of passing it into `iUnlockPayment` as an argument.  (not sure about this thought either.)

Reply:
removed